src/complex/Pipeline/PipelineFilter.cpp

```diff
@@ -241,10 +251,18 @@ bool PipelineFilter::execute(DataStructure& data, const std::atomic_bool& should
 
   IFilter::MessageHandler messageHandler{[this](const IFilter::Message& message) { this->notifyFilterMessage(message); }};
 
-  IFilter::ExecuteResult result = m_Filter->execute(data, getArguments(), this, messageHandler, shouldCancel);
-  m_PreflightValues = std::move(result.outputValues);
+  IFilter::ExecuteResult result;
+  if(m_Filter != nullptr)
+  {
+    result = m_Filter->execute(data, getArguments(), this, messageHandler, shouldCancel);
+    m_Warnings = result.result.warnings();
+  }
+  else
+  {
+    m_Warnings.push_back(Warning{-11, "This filter is just a placeholder! The original filter could not be found. See the filter comments for more details."});
```

Comment:
@mmarineBlueQuartz Yeah I wasn't sure what should actually happen in this case (error, warning, or nothing at all) and @imikejackson said to "Throw a warning at worst just stating that this is a place holder filter for another filter that is missing from the pipeline". I suppose since having a filter labeled "Unknown Filter" is pretty self explanatory and 9 times out of 10 it will cause other preflight errors anyway. But I'm fine changing this to be more restrictive if that's what we want.